internal/rvtools/parser.go

```diff
@@ -0,0 +1,139 @@
+package rvtools
+
+import (
+	"bytes"
+	"fmt"
+	"slices"
+
+	api "github.com/kubev2v/migration-planner/api/v1alpha1"
+	"github.com/xuri/excelize/v2"
+	"go.uber.org/zap"
+)
+
+func ParseRVTools(rvtoolsContent []byte) (*api.Inventory, error) {
+	excelFile, err := excelize.OpenReader(bytes.NewReader(rvtoolsContent))
+	if err != nil {
+		return nil, fmt.Errorf("error opening Excel file: %v", err)
+	}
+	defer excelFile.Close()
+	
+	sheets := excelFile.GetSheetList()
+
+	inventory := api.Inventory{}
+
+	inventory.Infra.Datastores = []api.Datastore{}
+	inventory.Infra.HostsPerCluster = []int{}
+	inventory.Vms.MigrationWarnings = api.MigrationIssues{}
+	inventory.Vms.NotMigratableReasons = []struct {
+		Assessment string `json:"assessment"`
+		Count      int    `json:"count"`
+		Label      string `json:"label"`
+	}{}
+	
+	if slices.Contains(sheets, "vInfo") {
+		rows, err := excelFile.GetRows("vInfo")
+		if err != nil {
+			zap.S().Infof("	Could not read vInfo sheet: %v", err)
+		} else {
+			vcenterUUID, _ := extractVCenterUUID(excelFile, "vInfo")
+			inventory.Vcenter.Id = vcenterUUID
+			vms, err := processVMInfo(rows)
+			if err != nil {
+				zap.S().Infof("Warning: VM processing failed: %v", err)
+			} else if len(vms) > 0 {
+				fillInventoryWithVMData(vms, &inventory)
+			}
+		}
+	}
+	
+	if slices.Contains(sheets, "vHost") {
+		rows, err := excelFile.GetRows("vHost")
+		if err != nil {
+			zap.S().Infof("Warning: Could not read vHost sheet: %v", err)
+			inventory.Infra.TotalHosts = 0
+			inventory.Infra.TotalClusters = 0
+			inventory.Infra.HostsPerCluster = []int{}
+			inventory.Infra.HostPowerStates = map[string]int{}
+		} else {
+			err = processHostInfo(rows, &inventory)
+			if err != nil {
+				zap.S().Infof("Warning: Host processing failed: %v", err)
+				inventory.Infra.TotalHosts = 0
+				inventory.Infra.TotalClusters = 0
+				inventory.Infra.HostsPerCluster = []int{}
+				inventory.Infra.HostPowerStates = map[string]int{}
+			}
+		}
+	}
```

Comment:
nit:
Use internal function to avoid repeated code:
```
func resetHostInfo(inventory *Inventory) {
    inventory.Infra.TotalHosts = 0
    inventory.Infra.TotalClusters = 0
    inventory.Infra.HostsPerCluster = []int{}
    inventory.Infra.HostPowerStates = map[string]int{}
}
```